blueocean-dashboard/src/main/js/components/MultiBranch.jsx

```diff
@@ -13,40 +14,34 @@ import { pipelineBranchesUnsupported } from './PipelinePage';
 
 const { object, array, func, string, any } = PropTypes;
 
-const EmptyState = ({ repoName }) => (
+const EmptyState = ({ repoName, t }) => (
     <main>
         <EmptyStateView iconName="branch">
-            <h1>Branch out</h1>
-
-            <p>
-                Create a branch in the repository <em>{repoName}</em> and
-                Jenkins will start testing your changes.
-            </p>
-
-            <p>
-                Give it a try and become a hero to your team.
-            </p>
-
-            <button>Enable</button>
+            <Markdown>
+                {t('EmptyState.branches', { 0: repoName })}
+            </Markdown>
```

Comment:
I think markdown is needed here since the alternative is creating 4/5 keys and wire them in a really ugly way. I am using markdown and not dangerouslySetInnerHTML to prevent xss attacks via i18n keys. 

I further think that we may want to use markdown in general much more since this would make it easy to create a markdown2pdf parser in the future.